shared/eval/src/tasks/mmlu_pro.rs

```diff
@@ -93,6 +93,15 @@ impl GenerateUntilTask for MMLUPro {
         });
         fewshot_documents
     }
+
+    fn get_stop_tokens(&self) -> Vec<String> {
+        vec!["Question:".to_string()]
```

Comment:
I am not 100% sure, but this should not be "\nQuestion:" ?
Check it with lm_evals please

Reply:
In `tasks/mmlu_pro/_default_template_yaml` they have:

```
generation_kwargs:
  until:
    - "Question:"
```

So unless they're adding the newline somewhere else, I think it's without. I can check with/without later in the benchmark perhaps.

Reply:
good!